compose/config/config_schema_v2.1.json

```diff
@@ -107,6 +107,8 @@
         "cpu_shares": {"type": ["number", "string"]},
         "cpu_quota": {"type": ["number", "string"]},
         "cpu_period": {"type": ["number", "string"]},
+        "cpu_rt_period": {"type": ["number", "string"]},
+        "cpu_rt_runtime": {"type": ["number", "string"]},
```

Comment:
I know I probably said something else the other day, but these options aren't compatible with v2.1 - they were added in API 1.25, which is 2.2+